### PR TITLE
feat(plugins): Introduce roadrunner plugin

### PR DESCRIPTION
**Pull request by darkweak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -787,6 +787,77 @@ func main() {
 }
 ```
 
+### Roadrunner middleware
+To use Souin as roadrunner middleware, you have to build your `rr` binary with the souin dependency.
+```toml
+[velox]
+build_args = ['-trimpath', '-ldflags', '-s -X github.com/roadrunner-server/roadrunner/v2/internal/meta.version=v2.10.7 -X github.com/roadrunner-server/roadrunner/v2/internal/meta.buildTime=10:00:00']
```
Comment (rustatian) [inline README.md:794]: You may use env variables to configure version, buildTime, and tokens, like:
`build_args = ['-trimpath', '-ldflags', '-s -X github.com/roadrunner-server/roadrunner/v2/internal/meta.version=${VERSION} -X github.com/roadrunner-server/roadrunner/v2/internal/meta.buildTime=${TIME}']`

And then use:

```bash
VERSION=v2.10.7 TIME=(date +%H:%M) vx build -c velox.toml -o "."
```

**Review by rustatian — APPROVED** [review]
lgtm 👍🏻

**Comment by rustatian** [comment]
After release, I'll include your plugin in the RR's main repository. Thank you very much ❤️

**Comment by darkweak** [comment]
The roadrunner tests fail because it doesn't handle the replace go.mod directive but everything works properly (at least unit tests ensure that).

**Pull request merged**